README.md

```diff
@@ -75,7 +75,7 @@ Finally, open the Xcode workspace: `{APP_Name}.xcworkspace`.
 
 ### 3. Create a Cloudant service instance
 
-Use the [IBM Cloud Catalog](https://console.bluemix.net/catalog/) to [create a Cloudant service instance](https://console.bluemix.net/catalog/services/cloudant) on the Lite plan.
+Use the [IBM Cloud Catalog](https://console.bluemix.net/catalog/) to [create a Cloudant service instance](https://console.bluemix.net/catalog/services/cloudant) on the Lite plan. Make sure to select **Use both legacy credentials and IAM** to receive the proper credential set for the Cloudant SDK.
```

Comment:
This option is going away fairly soon - why do we need to push people to get username & passwords, instead of focusing on IAM support?

Reply:
@dolph Is this announced somewhere?  I have not heard any announcement about end of life for legacy Cloudant credentials.  The Cloudant SDK for Swift only supports legacy authentication.  The Cloudant team does not have any plans to upgrade it until End of Life is announced.